screen-wake-lock/wakelock-type.https.window.js

```diff
@@ -1,21 +0,0 @@
-// META: script=/resources/testdriver.js
```

Comment:
Why were these tests rewritten? I generally prefer using `window.js` tests when testing Javascipt APIs at least in part because tools like `clang-format` know how to format `.js` files and not `.html` files.

Reply:
See https://github.com/web-platform-tests/wpt/pull/36122#issuecomment-1261580584